sceptre/plan/plan.py

```diff
@@ -53,134 +61,134 @@ def _generate_launch_order(self):
 
     def template(self, *args):
         """
-        Returns the CloudFormation template used to create the stack.
+        Returns the CloudFormation Template used to create the Stack.
 
-        :returns: The stack's template.
+        :returns: The Stack's Template.
```

Comment:
This will be an awkward update - but these methods will all now return a `dict` of `Stack` -> `<result>` (e.g. `str` for `template()`).